_postgres/queries_psycopg.py

```diff
@@ -299,3 +307,74 @@ def get_person(conn, id):
             } for mov in directed_rows
         ],
     })
+
+
+def update_movie(conn, id):
+    with conn.cursor() as cur:
+        cur.execute('''
+            UPDATE
+                movies
+            SET
+                title = movies.title || '---' || %(suffix)s
+            WHERE
+                movies.id = %(id)s
+            RETURNING
+                movies.id, movies.title
+        ''', dict(id=id, suffix=str(id)[:8]))
+
+        rows = cur.fetchall()
+        cur.connection.commit()
+
+    return json.dumps({
+        'id': rows[0][0],
+        'title': rows[0][1],
+    })
+
+
+def insert_user(conn, val):
+    num = random.randrange(1_000_000)
+    with conn.cursor() as cur:
+        cur.execute('''
+            INSERT INTO users (name, image) VALUES
```

Comment:
Can we also exercise a multi-link insert/update?  E.g do `insert_movie` with actors or whatnot.  I feel like the triviality of this DML doesn't reveal the complete picture.

Reply:
I'll add two more complex insert tests:
1) Actual nested insert of movie + director + actors, all of which would be new objects.
2) Inserting a new movie with pre-existing director and actors linked to it (the cost of fetching the pre-existing IDs would not be part of the test, just actually creating the appropriate base + multi-links).

The directors and actors links are many-to-many and will be a good stress test that's actually different from just writing an ID in a column for a 1-to-many case. Because if all you do is write more foreign keys in the same table, it's no different from writing other scalar columns in terms of what the expected performance might be.